### DISCUSSION TITLE
Rasberry Pi 4B Audio issue fixed

### DISCUSSION THREAD

**Discussion by musicaltank** [vicwomg/pikaraoke#261] [General]
After installing from the pikaraoke image, I was having trouble setting my default audio output on my RPi 4B to the headphone jack - audio was playing through HDMI only. I tried running "sudo raspi-config" and navigating through the menu options to force audio output to headphones (3.5 mm jack), but it always defaulted back to HDMI

So, I updated raspi-config and tried again - the menu structure changed, moving the audio settings to a different location, but I still could not force default output to headphones.

I eventually found this thread, and the solution at the bottom worked for me:
https://forums.raspberrypi.com/viewtopic.php?t=226849

To summarize:
Create/edit the system-wide alsa configuration file at /etc/asound.conf, e.g. with 'sudo nano /etc/asound.conf'
Into this file, paste:

pcm.!default {
type hw
card 0
}

ctl.!default {
type hw
card 0
}

To browse the normal Raspbian system files to verify config file location (I'm OCD that way), I first had to enter "cd /" into the command line, and then when I ran "ls" I could see the directories and navigate from there.

Hopefully this helps someone else struggling with the same issue.
